docs/schema/guidance/repr-beneficial-ownership.rst

```diff
@@ -0,0 +1,75 @@
+.. _representing-bo:
+
+Representing beneficial ownership
+========================================
+
+.. highlights::
+
+    **Key requirement:** If a person is a beneficial owner of an entity - whether directly or indirectly - and the person or entity is required to declare this beneficial ownership, there MUST be an Ownership-or-control Statement connecting the two which represents the beneficial ownership relationship.
+
+
+Overview
+------------------------
+
+Beneficial owners can exercise ownership-or-control *directly* in a company (entity) or *indirectly*, via intermediary entities and arrangements. It must be clear in a BODS dataset which people are declared as beneficial owners of which companies (and which companies are intermediaries). And it must be clear what control or ownership beneficial owners ultimately have (whether it is direct or indirect).
+
+.. figure:: ../../_assets/RepresentingChainsBODS-RealWorld.svg
+   :alt: Person 1 indirectly holds a 15 percent shareholding in Company E, via an intermediary: Company A. Person 2 directly holds 32 precent of Company E's shares.
+   :width: 500px
+   :align: center
+
+In the above situation, Person 1 *indirectly* benefits from a 15% shareholding in Company E and Person 2 has a *direct* 32% shareholding in Company E. In a jurisdiction where people with a shareholding (direct or indirect) of over 10% in a company should be disclosed as beneficial owners, both Person 1 and Person 2’s interests would be declared. Additionally, the jurisdiction may require that details of Person 1’s indirect interest are disclosed. That is: that some details of the chain Company E - Company A - Person 1 are also disclosed. 
+
+In BODS, the following properties are used to represent such information disclosure:
+
+* ``beneficialOwnershipOrControl`` (See :ref:`schema-interest`)
+* ``directOrIndirect`` (See :ref:`schema-interest`)
+* ``componentStatementIDs`` (See :ref:`schema-ownership-or-control-statement`)
+* ``isComponent`` (See :ref:`schema-entity-statement`)
+
+Requirements
+------------------------
+
+Representing beneficial ownership
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If a person is a beneficial owner of an entity, E, whether directly or indirectly, and one of them is required to declare this beneficial ownership:
```

Comment:
Maybe consider changing this to

"If a person is a beneficial owner of an entity (Company E in our example), whether directly or indirectly, "

Reply:
See comment below

Reply:
Yes, your wording is better than my suggestion.